### ISSUE TITLE
Accessibility issues in maps navigational menu

### ISSUE THREAD

**Issue by renujain31** [ushahidi/platform#4187]
**Describe the bug**
The drop down menu for options in posts is not accessible via keyboard.

**Where was the bug observed**
/views/map in v5 deployment

**Screenshots**
![Ushahidi Platform v5](https://user-images.githubusercontent.com/70752431/102888377-a7fee500-447e-11eb-8629-d56f9fd3c4b7.gif)
